scripts/cxg_admin_scripts/reprocess_datafile.py

```diff
@@ -30,9 +30,58 @@ def get_happy_stack_name(deployment) -> str:
     return f"{deployment}-{deployment}stack"
 
 
-def cxg_remaster(ctx):
+def cxg_remaster():
```

Comment:
is this related to this PR?

Reply:
ah, no--this should've been in a diff changelog. Cleaning up

Reply:
resolved